nodes/signal_shutdown.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# Simple script for shutting down the raspberry Pi at the press of a button.
+# by Inderpreet Singh
+# Adapted by Jake Rye and Bob Davis
+
+import RPi.GPIO as GPIO
```

Comment:
What if we use [Python Periphery](https://github.com/vsergeev/python-periphery), rather than `RPi` for GPIO access? This would avoid an explicit RPi dependency. It also happens to be the dependency we will probably use for I2C access if we get the sensors talking directly to the Pi.

A while back, I made a PR to include `python-periphery` in rosdep as `python-periphery-pip` https://github.com/ros/rosdistro/pull/14364. If we go this route, we can add it to `package.xml` as `<run_depend>python-periphery-pip</run_depend>`.

WDYT?